tg_odesli_bot/platforms.py

```diff
@@ -96,3 +96,12 @@ class AppleMusicPlatform(PlatformABC):
     url_re = r'https?://([a-zA-Z\d-]+\.)*music\.apple\.com/[^\s.,]*'
     name = 'Apple Music'
     order = 7
+
+
+class TidalPlatform(PlatformABC):
+    """Tidal platform."""
+
+    key = 'tidal'
+    url_re = r'https?://(www\.)?tidal\.com(/browse)?/track/\d+'
```

Comment:
Thanks for your contribution! 
I don't have a Tidal account to check it myself, but Tidal URLs on Odesli.co look like `https://listen.tidal.com/track/60635243`. May be this `listen` part should be added to regex?
```suggestion
    url_re = r'https?://(www\.|listen\.)?tidal\.com(/browse)?/track/\d+'
```

Reply:
Sure, that'd be perfect. I didn't see that before. Good catch!